app/controllers/tags_controller.rb

```diff
@@ -80,23 +80,23 @@ def destroy
     else
       flash[:success] = "Tag deleted."
 
-      url_params = {}
-      url_params[:page] = page if params[:page].present?
-      url_params[:view] = params[:view] if params[:view].present?
-      redirect_to tags_path(url_params)
+      permitted_params = {}
+      permitted_params[:page] = page if params[:page].present?
+      permitted_params[:view] = params[:view] if params[:view].present?
+      redirect_to tags_path(permitted_params)
```

Comment:
This looks like over-eager find-replace (right?)

```suggestion
      url_params = {}
      url_params[:page] = page if params[:page].present?
      url_params[:view] = params[:view] if params[:view].present?
      redirect_to tags_path(url_params)
```